astropy/cosmology/parameter/tests/test_parameter.py

```diff
@@ -486,3 +455,23 @@ def test_Parameter_repr_roundtrip(self, param):
         NP = eval(repr(P))  # Evaluate string representation back into a param.
 
         assert P == NP
+
+    # ========================================================================
+
```

Comment:
What are these lines good for?

Reply:
I like to separate unit tests from integration tests. So the above tests are for a specific method, e.g. ``test_Parameter_repr`` for ``repr``. This test is about a "real life" usage example.

Reply:
If this is deliberate and has meaning for you then it can stay.